eq-author/src/App/ErrorBoundary/index.test.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { shallow } from "enzyme";
+
+import ErrorBoundary from "./";
+import { ERR_PAGE_NOT_FOUND } from "constants/error-codes";
+import NotFound from "App/NotFoundPage";
+
+const MockComponent = () => null;
+
+describe("ErrorBoundary", () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = shallow(
+      <ErrorBoundary>
+        <MockComponent />
+      </ErrorBoundary>
+    );
+  });
+
+  it("should render the child component by default", () => {
+    expect(wrapper.instance().render()).toMatchObject(<MockComponent />);
```

Comment:
Could this be something like this?
`expect(wrapper.find(MockComponent)).toHaveLength(1)`

Reply:
yes. code updated